openshift-knative-operator/hack/update-manifests.sh

```diff
@@ -36,4 +37,7 @@ function download {
 }
 
 download serving $KNATIVE_SERVING_VERSION "${serving_files[@]}"
-download eventing $KNATIVE_EVENTING_VERSION "${eventing_files[@]}"
\ No newline at end of file
+# TODO: Remove this patch once 0.18.5 of serving or newer is available.
+git apply "$root/openshift-knative-operator/hack/001-liveness.patch"
```

Comment:
so the liveness patch existed previously (according to the diff above), but was not applied until yet ?

Reply:
No, it's new. Misreading the diff?

Reply:
ah, watching a highlighted patch in a git diff with the same highlighting confused me (i.e. the "red" is an "inner red" ;-)